### DISCUSSION TITLE
How to run Budibase with Podman?

### DISCUSSION THREAD

**Discussion by aiarchitect** [Budibase/budibase#6810] [General]
How to run Budibase with Podman? 
I tried starting the container in Fedora using podman, but budibase wouldnt start. The container runs, but the app doesnt start and running the healthcheck script inside shows that 3 services are not running! any help is appreciated.

**Comment by jonnymccullagh** [comment]
Hi @aiarchitect, I tested the single image build using Podman on AlmaLinux 8 so could be similar to your use-case. Can I check the process you followed? 
- Did you use the [instructions here](https://github.com/Budibase/budibase/tree/develop/hosting/single) to build the image ? or,
- Did you pull the image from the docker registry e.g. `docker run -d -t --name=budibase -p 10000:80 -v /local/path/data:/data --restart unless-stopped budibase/budibase:latest` ?
- Did you set all the needed environment variables?
- Can you check the logs from the container (`podman logs budibase`) for error messages ?

If you can provide info on the steps you took and commands you ran (with sensitive variables redacted) I will attempt to replicate on AlmaLinux or Fedora.

**Reply by aiarchitect** [reply]
Hi @jonnymccullagh ,

Thanks for looking into this issue. 

I did not build the image locally, should I build the image?

Please find the command that I used to start the container, 

podman run \
-p 7764:80 \
-v ~/lab/test/budibase/data:/data \
-e INTERNAL_API_KEY=abcd123 \
-e JWT_SECRET=abcd321 \
-e MINIO_ROOT_USER=miniouser \
-e MINIO_ROOT_PASSWORD=Install@123 \
-e REDIS_PASSWORD=abcd1212 \
-e COUCHDB_USER=couchuser \
-e COUCHDB_PASSWORD=Install@123 \
--name crmapp \
-d docker.io/budibase/budibase:latest

So far I have tried in Fedora and Ubuntu using Podman.

Logs:
Restarting nginx: nginx16:C 20 Jul 2022 09:11:47.403 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
16:C 20 Jul 2022 09:11:47.403 # Redis version=6.0.16, bits=64, commit=00000000, modified=0, pid=16, just started
16:C 20 Jul 2022 09:11:47.403 # Configuration loaded
16:M 20 Jul 2022 09:11:47.452 * Running mode=standalone, port=6379.
16:M 20 Jul 2022 09:11:47.452 # Server initialized
16:M 20 Jul 2022 09:11:47.452 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
16:M 20 Jul 2022 09:11:47.458 * Ready to accept connections
WARNING: MINIO_ACCESS_KEY and MINIO_SECRET_KEY are deprecated.
         Please use MINIO_ROOT_USER and MINIO_ROOT_PASSWORD
Formatting 1st pool, 1 set(s), 1 drives per set.
WARNING: Host local has more than 0 drives of set. A host failure will result in data becoming unavailable.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
.
Restarting nginx: nginx
 You are running an older version of MinIO released 2 days ago 
 Update: Run `mc admin update` 


MinIO Object Storage Server
Copyright: 2015-2022 MinIO, Inc.
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Version: RELEASE.2022-07-15T03-44-22Z (go1.18.4 linux/amd64)

Status:         1 Online, 0 Offline. 
API: http://10.0.2.100:9000  http://127.0.0.1:9000     
Console: http://10.0.2.100:41335 http://127.0.0.1:41335   

Documentation: https://docs.min.io
[info] 2022-07-20T09:11:49.507071Z couchdb@127.0.0.1 <0.256.0> -------- Preflight check: Asserting Admin Account

[info] 2022-07-20T09:11:49.512440Z couchdb@127.0.0.1 <0.256.0> -------- Apache CouchDB 3.2.1 is starting.

[info] 2022-07-20T09:11:49.512615Z couchdb@127.0.0.1 <0.257.0> -------- Starting couch_sup
[notice] 2022-07-20T09:11:49.530492Z couchdb@127.0.0.1 <0.110.0> -------- config: [admins] 9d7365a97ded4da286631930f3f33139 set to '****' for reason nil
[info] 2022-07-20T09:11:49.624740Z couchdb@127.0.0.1 <0.256.0> -------- Apache CouchDB has started. Time to relax.

[notice] 2022-07-20T09:11:49.641203Z couchdb@127.0.0.1 <0.298.0> -------- rexi_server : started servers
[notice] 2022-07-20T09:11:49.644415Z couchdb@127.0.0.1 <0.302.0> -------- rexi_buffer : started servers
.
/app /
[warning] 2022-07-20T09:11:49.704204Z couchdb@127.0.0.1 <0.310.0> -------- creating missing database: _nodes
[info] 2022-07-20T09:11:49.704350Z couchdb@127.0.0.1 <0.311.0> -------- open_result error {not_found,no_db_file} for _nodes
[info] 2022-07-20T09:11:49.841894Z couchdb@127.0.0.1 <0.328.0> -------- open_result error {not_found,no_db_file} for _dbs
[warning] 2022-07-20T09:11:49.841925Z couchdb@127.0.0.1 <0.326.0> -------- creating missing database: _dbs
[warning] 2022-07-20T09:11:49.841989Z couchdb@127.0.0.1 <0.325.0> -------- creating missing database: _dbs
[notice] 2022-07-20T09:11:49.870005Z couchdb@127.0.0.1 <0.340.0> -------- mem3_reshard_dbdoc start init()
[notice] 2022-07-20T09:11:49.880291Z couchdb@127.0.0.1 <0.342.0> -------- mem3_reshard start init()
[notice] 2022-07-20T09:11:49.880490Z couchdb@127.0.0.1 <0.343.0> -------- mem3_reshard db monitor <0.343.0> starting
[notice] 2022-07-20T09:11:49.887906Z couchdb@127.0.0.1 <0.342.0> -------- mem3_reshard starting reloading jobs
[notice] 2022-07-20T09:11:49.888422Z couchdb@127.0.0.1 <0.342.0> -------- mem3_reshard finished reloading jobs
[info] 2022-07-20T09:11:49.972251Z couchdb@127.0.0.1 <0.349.0> -------- Apache CouchDB has started. Time to relax.

[info] 2022-07-20T09:11:49.972422Z couchdb@127.0.0.1 <0.349.0> -------- Apache CouchDB has started on http://any:5984/
[notice] 2022-07-20T09:11:49.998992Z couchdb@127.0.0.1 <0.372.0> -------- chttpd_auth_cache changes listener died because the _users database does not exist. Create the database to silence this notice.
[error] 2022-07-20T09:11:49.999649Z couchdb@127.0.0.1 emulator -------- Error in process <0.373.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

[error] 2022-07-20T09:11:50.000100Z couchdb@127.0.0.1 emulator -------- Error in process <0.373.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

[notice] 2022-07-20T09:11:50.134064Z couchdb@127.0.0.1 <0.420.0> -------- Missing system database _users

                        -------------

__/\\\\\\\\\\\\\____/\\\\____________/\\\\____/\\\\\\\\\_____
 _\/\\\/////////\\\_\/\\\\\\________/\\\\\\__/\\\///////\\\___
  _\/\\\_______\/\\\_\/\\\//\\\____/\\\//\\\_\///______\//\\\__
   _\/\\\\\\\\\\\\\/__\/\\\\///\\\/\\\/_\/\\\___________/\\\/___
    _\/\\\/////////____\/\\\__\///\\\/___\/\\\________/\\\//_____
     _\/\\\_____________\/\\\____\///_____\/\\\_____/\\\//________
      _\/\\\_____________\/\\\_____________\/\\\___/\\\/___________
       _\/\\\_____________\/\\\_____________\/\\\__/\\\\\\\\\\\\\\\_
        _\///______________\///______________\///__\///////////////__


                          Runtime Edition

        PM2 is a Production Process Manager for Node.js applications
                     with a built-in Load Balancer.

                Start and Daemonize any application:
                $ pm2 start app.js

                Load Balance 4 instances of api.js:
                $ pm2 start api.js -i 4

                Monitor in production:
                $ pm2 monitor

                Make pm2 auto-boot at server restart:
                $ pm2 startup

                To go further checkout:
                http://pm2.io/


                        -------------

[PM2] Spawning PM2 daemon with pm2_home=/root/.pm2
[PM2] PM2 Successfully daemonized
[PM2] Starting /bin/bash in fork_mode (1 instance)
[PM2] Done.
┌─────┬────────┬─────────────┬─────────┬─────────┬──────────┬────────┬──────┬───────────┬──────────┬──────────┬──────────┬──────────┐
│ id  │ name   │ namespace   │ version │ mode    │ pid      │ uptime │ ↺    │ status    │ cpu      │ mem      │ user     │ watching │
├─────┼────────┼─────────────┼─────────┼─────────┼──────────┼────────┼──────┼───────────┼──────────┼──────────┼──────────┼──────────┤
│ 0   │ app    │ default     │ N/A     │ fork    │ 193      │ 0s     │ 0    │ online    │ 0%       │ 23.4mb   │ root     │ disabled │
└─────┴────────┴─────────────┴─────────┴─────────┴──────────┴────────┴──────┴───────────┴──────────┴──────────┴──────────┴──────────┘
/
/worker /
[PM2] Starting /bin/bash in fork_mode (1 instance)
[PM2] Done.
┌─────┬───────────┬─────────────┬─────────┬─────────┬──────────┬────────┬──────┬───────────┬──────────┬──────────┬──────────┬──────────┐
│ id  │ name      │ namespace   │ version │ mode    │ pid      │ uptime │ ↺    │ status    │ cpu      │ mem      │ user     │ watching │
├─────┼───────────┼─────────────┼─────────┼─────────┼──────────┼────────┼──────┼───────────┼──────────┼──────────┼──────────┼──────────┤
│ 0   │ app       │ default     │ N/A     │ fork    │ 193      │ 0s     │ 0    │ online    │ 150%     │ 73.2mb   │ root     │ disabled │
│ 1   │ worker    │ default     │ N/A     │ fork    │ 211      │ 0s     │ 0    │ online    │ 0%       │ 15.7mb   │ root     │ disabled │
└─────┴───────────┴─────────────┴─────────┴─────────┴──────────┴────────┴──────┴───────────┴──────────┴──────────┴──────────┴──────────┘
/
[notice] 2022-07-20T09:11:55.000823Z couchdb@127.0.0.1 <0.372.0> -------- chttpd_auth_cache changes listener died because the _users database does not exist. Create the database to silence this notice.
[error] 2022-07-20T09:11:55.000965Z couchdb@127.0.0.1 emulator -------- Error in process <0.522.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

[error] 2022-07-20T09:11:55.001032Z couchdb@127.0.0.1 emulator -------- Error in process <0.522.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

[notice] 2022-07-20T09:11:55.027423Z couchdb@127.0.0.1 <0.394.0> -------- couch_replicator_clustering : cluster stable
[notice] 2022-07-20T09:11:55.032942Z couchdb@127.0.0.1 <0.404.0> -------- Started replicator db changes listener <0.523.0>
[info] 2022-07-20T09:11:55.033884Z couchdb@127.0.0.1 <0.525.0> -------- open_result error {not_found,no_db_file} for _replicator
[notice] 2022-07-20T09:12:00.001866Z couchdb@127.0.0.1 <0.372.0> -------- chttpd_auth_cache changes listener died because the _users database does not exist. Create the database to silence this notice.
[error] 2022-07-20T09:12:00.002336Z couchdb@127.0.0.1 emulator -------- Error in process <0.577.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

[error] 2022-07-20T09:12:00.002833Z couchdb@127.0.0.1 emulator -------- Error in process <0.577.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

[notice] 2022-07-20T09:12:04.641266Z couchdb@127.0.0.1 <0.298.0> -------- rexi_server : cluster stable
[notice] 2022-07-20T09:12:04.642684Z couchdb@127.0.0.1 <0.302.0> -------- rexi_buffer : cluster stable
[notice] 2022-07-20T09:12:05.005272Z couchdb@127.0.0.1 <0.372.0> -------- chttpd_auth_cache changes listener died because the _users database does not exist. Create the database to silence this notice.
[error] 2022-07-20T09:12:05.005480Z couchdb@127.0.0.1 emulator -------- Error in process <0.610.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

[error] 2022-07-20T09:12:05.005553Z couchdb@127.0.0.1 emulator -------- Error in process <0.610.0> on node 'couchdb@127.0.0.1' with exit value:
{database_does_not_exist,[{mem3_shards,load_shards_from_db,"_users",[{file,"src/mem3_shards.erl"},{line,400}]},{mem3_shards,load_shards_from_disk,1,[{file,"src/mem3_shards.erl"},{line,375}]},{mem3_shards,load_shards_from_disk,2,[{file,"src/mem3_shards.erl"},{line,404}]},{mem3_shards,for_docid,3,[{file,"src/mem3_shards.erl"},{line,97}]},{fabric_doc_open,go,3,[{file,"src/fabric_doc_open.erl"},{line,39}]},{chttpd_auth_cache,ensure_auth_ddoc_exists,2,[{file,"src/chttpd_auth_cache.erl"},{line,198}]},{chttpd_auth_cache,listen_for_changes,1,[{file,"src/chttpd_auth_cache.erl"},{line,145}]}]}

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0[error] 2022-07-20T09:12:06.173971Z couchdb@127.0.0.1 <0.594.0> 8d9c17ab4d Request to create N=3 DB but only 1 node(s)
{"ok":true}
100    12  100    12    0     0     16      0 --:--:-- --:--:-- --:--:--    16
[notice] 2022-07-20T09:12:06.585848Z couchdb@127.0.0.1 <0.594.0> 8d9c17ab4d localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 PUT /_users 201 ok 630
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0[error] 2022-07-20T09:12:06.945615Z couchdb@127.0.0.1 <0.611.0> 58a76c66d3 Request to create N=3 DB but only 1 node(s)
100    12  100    12    0     0     56      0 --:--:-- --:--:-- --:--:--    56
{"ok":true}
[notice] 2022-07-20T09:12:07.122571Z couchdb@127.0.0.1 <0.611.0> 58a76c66d3 localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 PUT /_replicator 201 ok 178
[notice] 2022-07-20T09:12:07.127948Z couchdb@127.0.0.1 <0.523.0> -------- creating replicator ddoc <<"shards/80000000-ffffffff/_replicator.1658308326">>
[notice] 2022-07-20T09:12:25.005273Z couchdb@127.0.0.1 <0.859.0> b1c1939801 localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-db/ 404 ok 120
[error] 2022-07-20T09:12:25.036183Z couchdb@127.0.0.1 <0.860.0> 3396963bdb Request to create N=3 DB but only 1 node(s)
[notice] 2022-07-20T09:12:25.092246Z couchdb@127.0.0.1 <0.860.0> 3396963bdb localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 PUT /global-db/ 201 ok 58
[notice] 2022-07-20T09:12:25.101669Z couchdb@127.0.0.1 <0.861.0> 4a3031d1a5 localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-db/migrations? 404 ok 2
[notice] 2022-07-20T09:12:25.129703Z couchdb@127.0.0.1 <0.862.0> 7d818732ab localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-info/ 404 ok 2
[notice] 2022-07-20T09:12:31.765492Z couchdb@127.0.0.1 <0.942.0> 7b414446dc localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-db/ 200 ok 46
[notice] 2022-07-20T09:12:31.791974Z couchdb@127.0.0.1 <0.943.0> d5df789a96 localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-db/migrations? 404 ok 2
[notice] 2022-07-20T09:12:31.832656Z couchdb@127.0.0.1 <0.944.0> b6e32180ff localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-info/ 404 ok 3
[notice] 2022-07-20T09:12:38.418515Z couchdb@127.0.0.1 <0.1017.0> cf1aa88053 localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-db/ 200 ok 5
[notice] 2022-07-20T09:12:38.456482Z couchdb@127.0.0.1 <0.1018.0> edb76ca3fa localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-db/migrations? 404 ok 2
[notice] 2022-07-20T09:12:38.520927Z couchdb@127.0.0.1 <0.1019.0> 28b4bebeaa localhost:5984 127.0.0.1 9d7365a97ded4da286631930f3f33139 GET /global-info/ 404 ok 2

**Reply by aiarchitect** [reply]
![image](https://user-images.githubusercontent.com/13310387/179946845-c5e65862-ffd4-4402-9336-1ebd24877e29.png)

![image](https://user-images.githubusercontent.com/13310387/179947011-284d3797-d333-4a4a-bc85-c3b4028ba61f.png)

**Reply by jonnymccullagh** [reply]
This is great. I'm installing AlmaLinux now to test and will report back.

**Reply by jonnymccullagh** [reply]
I've just installed a fresh AlmaLinux (very similar to Fedora) with 4GB RAM, 32GB disk and 4 CPUs and ran:
```
dnf update
dnf install podman
```
I then disabled selinux and rebooted.
Then ran:
```
mkdir -p ~/lab/test/budibase/data
podman run -p 7764:80 -v ~/lab/test/budibase/data:/data -e INTERNAL_API_KEY=abcd123 -e JWT_SECRET=abcd321 -e MINIO_ROOT_USER=miniouser -e MINIO_ROOT_PASSWORD=Install@123 -e REDIS_PASSWORD=abcd1212 -e COUCHDB_USER=couchuser -e COUCHDB_PASSWORD=Install@123 --name crmapp -d docker.io/budibase/budibase:latest
```
It came up fine and is answering on port 7764.
I eentered the container:
```
podman exec -it crmapp bash
```
and ran the healthcheck:
```
./healthcheck.sh
```
And it returned fine too.
Can you prune your existing images and try to re-run it - in case you had a slightly older image version? And let us know your system resources? 
Failing that I am happy to jump on a Google Meet.

**Reply by aiarchitect** [reply]
@jonnymccullagh thanks your suggestion worked, pruned the existing latest image and re-run the command now I can see the login screen. 

![image](https://user-images.githubusercontent.com/13310387/180055345-ca362f58-31de-42ab-a689-89033097bebd.png)

**Comment by aiarchitect** [comment]
Issue resolved with @jonnymccullagh  suggestion : prune the existing image and re-run the command

podman run
-p 7764:80
-v ~/lab/test/budibase/data:/data
-e INTERNAL_API_KEY=abcd123
-e JWT_SECRET=abcd321
-e MINIO_ROOT_USER=miniouser
-e MINIO_ROOT_PASSWORD=Install@123
-e REDIS_PASSWORD=abcd1212
-e COUCHDB_USER=couchuser
-e COUCHDB_PASSWORD=Install@123
--name crmapp
-d docker.io/budibase/budibase:latest
